test_tools/process_client.py

```diff
@@ -789,3 +789,24 @@ def get_workspace_publication_thumbnail(publication_type, workspace, name, *, ac
     response = requests.get(r_url, headers=headers, timeout=HTTP_TIMEOUT)
     raise_layman_error(response)
     return response.content
+
+
+def get_users(headers=None):
+    headers = headers or {}
+    with app.app_context():
+        r_url = url_for('rest_users.get')
+    response = requests.get(r_url, headers=headers, timeout=HTTP_TIMEOUT)
+    response.raise_for_status()
+    return response.json()
+
+
+def delete_user(username, actor_name):
```

Comment:
```suggestion
def delete_user(username, *, actor_name):
```

This will change `actor_name` argument from positional argument to keyword argument. We are doing this because of readability, compare
```
delete_user('user1', 'user2')
```
vs.
```
delete_user('user1', actor_name='user2')
```